### PR TITLE
Remove google plus share button due to service has been declined 

### PR DESCRIPTION
**Pull request by AlexAndBear**
## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Screenshots (if appropriate):
Before:
image

After:
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [x] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by pmaier1** [comment]
Basically :+1: Did you try whether it really doesn't work anymore? There is some successor of Google+ https://en.wikipedia.org/wiki/Google_Currents

**Comment by JammingBen** [comment]
We also have buttons in the sidebar:

image

**Comment by JanAckermann** [comment]
@pmaier1 yes, told me that it isn't available for private and brand users anymore

**Comment by JanAckermann** [comment]
@JammingBen resolved

File: changelog/unreleased/38705 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Enhancement: Remove google+ share button from personal settings
+
+Due to the google+ service has been declined, we won't show the google+ share
+button anymore.
+
+https://github.com/owncloud/core/pull/38075
```
Comment (JammingBen) [inline changelog/unreleased/38705:6]:
```suggestion
https://github.com/owncloud/core/pull/38705
```

**Review by JammingBen — CHANGES_REQUESTED** [review]
Fix PR link in changelog, then 🚀

**Comment by JanAckermann** [comment]
@pmaier1 i would like to merge if you confirm.

**Review by JammingBen — APPROVED** [review]

**Review by pmaier1 — APPROVED** [review]

**Pull request merged**